contributor-docs/development/tests/conformance-tests-sonobuoy.md

```diff
@@ -0,0 +1,220 @@
+---
+title: SCS Conformance Test KaaS Sonobuoy
+type:
+status: Draft
+track: Global
+---
+
+SovereignCloudStack (SCS) makes use of [Sonobuoy][sonobuoy] as a test framework to run its tests on Kubernetes clusters, which are to be audited for compliance.
+The aim of using this framework is to make the execution of tests on a KaaS infrastructure as simple as possible.
+Hence this reduces the effort required to establish SCS conformity to a minimum.
+
+In short this is achieved by storing all tests in a container image, which can then be called and launched on the clusters managed by sonoobuoy.
+
+> A more detailed description of why the SCS has decided to use sonobuoys can be found on the corresponding [Decsision Record][sonbouy-decision-record].
+> In addition, sonobuoy is also used as the toolset for executing Kubernetes very own [conformance tests][k8s-conformance].
+
+
+This document is intended to assist conformance test authors to integrate their tests into the sonobuoy framework.
+This requires a user to write the conformance tests in Golang, as this is the language provided by the framework itself.
+
+
+# Step-by-step instructions for the development of sonobuoy tests using `docker` and `kind`:
+
+> This guide refers to the brief instructions provided in the [standards repository][scs-sonobuoy-example-guide]
+
+
+## Prerequisite
+
+* [docker][docker-installation]
+* [kind][kind-installation]
+* [sonobuoy][sonobuoy-installation] 
+
+```
+go install github.com/vmware-tanzu/sonobuoy@latest
+```
+
+## Set up development environment:
+
+
+1. Clone the standard repository containing the conformance tests and navigate to the Sonobuoy example located at `standards/Tests/kaas/kaas-sonobuoy-go-example-e2e-framework`
+
+
+```
```

Comment:
maybe add shell or something here

```suggestion
```shell
```

Reply:
same below

Reply:
I have now added an attribute for the language or CLI tool used to all fenced code blocks.